containers/DvmState.ts

```diff
@@ -39,25 +36,29 @@ const useContractState = () => {
   // get state from EMP
   const queryState = async () => {
     if (
-      dvm !== null &&
+      votingContract !== null &&
+      storeContract !== null &&
+      collAddress !== null &&
       priceIdentifier !== null &&
       expirationTimestamp !== null
     ) {
       const res = await Promise.all([
-        dvm.hasPrice(
+        votingContract.hasPrice(
           priceIdentifier.toString(),
           expirationTimestamp.toNumber(),
           { from: empAddress }
         ),
+        storeContract.computeFinalFee(collAddress, { from: empAddress }),
       ]);
 
       const hasPrice = res[0] as boolean;
+      const finalFee = parseFloat(fromWei(res[1][0].toString()));
```

Comment:
Does this still work if the collateral does not have a final fee in the store?

Reply:
i.e. does it return 0

Reply:
Yep, I think it returns 0 in that case: https://github.com/UMAprotocol/protocol/blob/acf99518ed0f42756814989331a8ed592ab99ebe/core/contracts/oracle/implementation/Store.sol#L132-L134

Reply:
it should,yes.